docs/source/contribution.rst

```diff
@@ -3,197 +3,64 @@
-it is the first documentation of your code that developers read, which is a very important part of your PR.
+You have to write a precise explanation of **what** and **how** in the description;
+this is the first documentation of your code and an important part of your PR.
 
-Once you send a PR, it is automatically tested on `Travis CI <https://travis-ci.org/chainer/chainer/>`_ for Linux and Mac OS X, and on `AppVeyor <https://ci.appveyor.com/project/chainer/chainer>`_ for Windows.
-Your PR needs to pass at least the test for Linux on Travis CI.
-After the automatic test passes, some of the core developers will start reviewing your code.
-Note that this automatic PR test only includes CPU tests.
+However, even if your code is not complete, you can send a PR as a *work-in-progress (WIP) PR* by prefixing the PR title with ``[WIP]``.
+If you just describe the PR, the core team and other contributors can join the discussion about how to proceed with it.
+WIP PRs may occasionally be be useful for discussing based on concrete code.
 
-.. note::
+When a PR is created (or updated), it is automatically tested in the CI environment.
+Once tests pass and the PR is approved by a reviewer from the core team, it will be merged.
 
-   We are also running continuous integration with GPU tests for the ``master`` branch and the versioned branch of the latest major version.
-   Since this service is currently running on our internal server, we do not use it for automatic PR tests to keep the server secure.
+.. note::
 
-If you are planning to add a new feature or modify existing APIs, **it is recommended to open an issue and discuss the design first.**
-The design discussion needs lower cost for the core developers than code review.
-Following the consequences of the discussions, you can send a PR that is smoothly reviewed in a shorter time.
+    If you are planning to add a new feature or modify existing APIs, **it is recommended to open an issue and discuss the design first.**
+    Following the consequences of the discussions, you can send a PR that is smoothly reviewed in a shorter time.
 
-Even if your code is not complete, you can send a pull request as a *work-in-progress PR* by putting the ``[WIP]`` prefix to the PR title.
-If you write a precise explanation about the PR, core developers and other contributors can join the discussion about how to proceed the PR.
-WIP PR is also useful to have discussions based on a concrete code.
+Issue/Pull Request Labels
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Issues and PRs are labeled on GitHub so that they can be grouped, filtered and better maintained.
+For instance, a label can indicate that a ticket needs response from the PR author, or that an issue needs immediate action in case of a critical bug.
+Please refer to the `list of lables on GitHub <https://github.com/chainer/chainer/labels>`_.
+**Note that revision releases cannot include PRs labeled as a feature or compability breaking.**
```

Comment:
Can't we remove this sentence? Recently we do not make revision releases, so almost all releases actually include feature addition (for compatibility breaks, it would be meaningful to mention that all minor releases cannot include them).

Reply:
Removed.